README.md

```diff
@@ -78,11 +78,6 @@ To do List Website is a project that allows a user to add to do tasks, display t
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍